Trying to import natural in TypeScript yields a linter error

If I do something like

```typescript
import { LevenshteinDistance } from "natural";
```

and try to type check my project I get this
```
node_modules/natural/lib/natural/ngrams/index.d.ts:25:23 - error TS7016: Could not find a declaration file for module '../tokenizers/tokenizer'. 'node_modules/natural/lib/natural/tokenizers/tokenizer.js' implicitly has an 'any' type.
```

Same thing happens here: `node_modules/natural/lib/natural/tfidf/index.d.ts:26:23`. It's importing the non-typed file.

I now have to turn on the `skipLibCheck` tsconfig flag for my linter checks to pass. I wouldn't want that.

Maybe just importing `Tokenizer` from the typed `lib/natural/tokenizers` will fix the issue

--- a/lib/natural/ngrams/index.d.ts
+++ b/lib/natural/ngrams/index.d.ts
@@ -22,7 +22,7 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 */
 
-import Tokenizer from '../tokenizers/tokenizer'
+import { Tokenizer } from '../tokenizers'
 
 declare interface NGramsStats {
   ngrams: string[][]

--- a/lib/natural/tfidf/index.d.ts
+++ b/lib/natural/tfidf/index.d.ts
@@ -23,7 +23,7 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 */
 
-import Tokenizer from '../tokenizers/tokenizer'
+import { Tokenizer } from '../tokenizers'
 
 declare type TfIdfCallback = (i: number, measure: number, key?: string) => void
 

